dynatrace/settings/services/cache/cache_test.go

```diff
@@ -1,3 +1,5 @@
+//go:build unit
+
```

Comment:
file is missing license header

Reply:
It's out of scope for this ticket, as all newly introduced files have a license header.
I could do a follow-up after this one is merged. This affects all files, not just the test files.